drivers/storage/vbox/storage/vbox_storage.go

```diff
@@ -73,6 +73,7 @@ func (d *driver) LocalDevices(
 	if ld, ok := context.LocalDevices(ctx); ok {
 		return ld, nil
 	}
+	panic("here1")
```

Comment:
Yes

On Thursday, May 12, 2016, Schley Andrew Kutz <notifications@github.com>
wrote:

> In drivers/storage/vbox/storage/vbox_storage.go
> <https://github.com/emccode/libstorage/pull/122#discussion_r63042498>:
>
> > @@ -73,6 +73,7 @@ func (d *driver) LocalDevices(
> >  	if ld, ok := context.LocalDevices(ctx); ok {
> >  		return ld, nil
> >  	}
> > +	panic("here1")
>
> Hi @clintonskitson <https://github.com/clintonskitson>,
>
> I'm assuming this was left in on accident?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/emccode/libstorage/pull/122/files/7df086b7cf1b2c7531b04bcdd8bbd037ff4cb294#r63042498>
>